Formula/partio.rb

```diff
@@ -18,28 +18,24 @@ class Partio < Formula
   depends_on "doxygen" => :build
   depends_on "swig" => :build
 
-  # These fixes are upstream and can be removed in the next released version.
-  patch do
-    url "https://github.com/wdas/partio/commit/5b80b00ddedaef9ffed19ea4e6773ed1dc27394e.diff?full_index=1"
-    sha256 "b14b5526d5b61a3dfec7ddd07b54d5a678170b15b3f83687ab1b4151ae0cd5f3"
-  end
-
-  patch do
-    url "https://github.com/wdas/partio/commit/bdce60e316b699fb4fd813c6cad9d369205657c8.diff?full_index=1"
-    sha256 "58dc0b77155b80301595c0b6a439e852f41779a31348f1716f2c9714273c638b"
-  end
-
-  patch do
-    url "https://github.com/wdas/partio/commit/e557c212b0e8e0c4830e7991541686d568853afd.diff?full_index=1"
-    sha256 "b20a25142316cf93c0cc1188508c39a01275cf784d628e9768d5ee1471adbee2"
+  resource "bgeo" do
+    url "https://raw.githubusercontent.com/wdas/partio/dc898054d9144a409fe0e3f8d7f957e1a50ed8a1/src/data/test.bgeo"
+    sha256 "b0fe4adcc043d59448cfe690389e45d0f49b6ead675b10d028ca3d70b85999e8"
   end
 
   def install
+    ENV["PYTHON_LIBS"] = "-undefined dynamic_lookup"
```

Comment:
@fxcoudert this did not work with:

```
failed: brew audit partio --online
Error: 1 problem in 1 formula detected
partio:
  * python modules have explicit framework links
    These python extension modules were linked directly to a Python
    framework binary. They should be linked with -undefined dynamic_lookup
    instead of -lpython or -framework Python.
      /usr/local/opt/partio/lib/python2.7/site-packages/_partio.so
```

Could you help me with it?